test/isIterable.js

```diff
@@ -0,0 +1,47 @@
+import * as RA from '../src';
+import eq from './shared/eq';
+
+const myCustomIterable = {
+  *[Symbol.iterator]() {
+    yield 1;
+    yield 2;
+    yield 3;
+  },
+};
+
+describe('isIterable', function() {
+  it('tests a value to determine if it is iterable', function() {
```

Comment:
Can we structure tests in following way ? Pls also use assertions from `chai` and not our custom `eq` function. We're trying to migrate  to more modern test structure lately; example here: https://github.com/char0n/ramda-adjunct/blob/master/test/allEqual.js

```js
describe('isIterable', function() {
  context('given value with iterable protocol', function() {
    specify('should return true', function() { 
      // all cases showing iterable protocol
    }); 
  });
  
  context('given value without iterable protocol', function() {
    specify('should return false', function() {
      // all cases showing non iterable protocol
    });
  });

  context('given custom iterable', function() {
    // define custom iterable here
    specify('should return true', function() {
      // test it
    });
  });

  it('should support placeholder to specify "gaps"', function() {
    const isIterable = RA.isIterable(R.__);

    assert.isTrue(isIterable([]));
  });
});

Reply:
You can also add case for `asyncIterable` and show that it's now failing.

Reply:
ahh, I was copying from somewhere else - must have picked the wrong thing to copy.  updating now.

Reply:
ok, I updated everything but I'm not quite sure exactly what you mean about adding a case `asyncIterable` failing.  Would you like me to add a test for that in the `isIterable` tests?  Seems like that should be a separate thing if/when that function is made.

Reply:
Lets avoid ssynciterable for now

Reply:
ok, so we're good to go on this PR then?